### PR TITLE
Move dnsmasq config file to /etc/dnsmasq.d/

### PR DESCRIPTION
**Pull request by jvonau**
### Fixes Bug
Bugfix don't supply named/bind files when not enabled.
### Description of changes proposed in this pull request.
move dnsmasq config file to /etc/dnsmasq.d/
### Smoke-tested in operating system.
not yet

### CODE REVIEW

File: roles/network/tasks/enable_services.yml [hunk 1]
```diff
@@ -50,9 +51,15 @@
 - name: Install /etc/dnsmasq.conf from template, if dnsmasq_enabled
```
Comment (holta) [inline roles/network/tasks/enable_services.yml]:
```suggestion
- name: Install /etc/dnsmasq.d/iiab.conf from template, if dnsmasq_enabled
```

Comment (holta) [inline roles/network/tasks/enable_services.yml]:
```suggestion
- name: Install /etc/dnsmasq.d/iiab.conf from template, if dnsmasq_enabled
```

File: roles/network/tasks/enable_services.yml [hunk 2]
```diff
@@ -50,9 +51,15 @@
...
   template:
     src: network/dnsmasq.conf.j2
-    dest: /etc/dnsmasq.conf
+    dest: /etc/dnsmasq.d/iiab.conf
   when: dnsmasq_enabled and dnsmasq_install
```
Comment (holta) [inline roles/network/tasks/enable_services.yml]:
```suggestion
  when: dnsmasq_enabled and dnsmasq_install and (iiab_network_mode != "Appliance")
```

Comment (jvonau) [reply roles/network/tasks/enable_services.yml]: won't that break the parsing when not aligned under the 'name'?

File: roles/network/tasks/enable_services.yml (hunk 1 above)
Comment (jvonau) [reply roles/network/tasks/enable_services.yml]: I don't see an extra space when viewing the file and was not changed by this PR

**Comment by holta** [comment]
Tested with @jvonau on a fresh Ubuntu Server 18.04.1 VM and with an upgrade on Raspbian Lite.

**Pull request merged**